62968 EDI - Tri-Lakes (Welch)

tyndmar opened the pull request:
Programs changed:
po/po-hrexp.p
- almost a complete rewrite
- driving issue was lack of reference to hrms reftable (removed as unneeded for valley)
- put this back, plus minor changes from conversations with EFI, Welch, TriLakes
sys/inc/hrms.i
- added reference to NK1 "HRMS" integer value
- must be set to "1" to use reftable; this to prevent unplanned changes to Valley transmissions

Commit:
62968 EDI - Tri-Lakes (Welch)

Programs changed:
po/po-hrexp.p
- almost a complete rewrite
- driving issue was lack of reference to hrms reftable (removed as unneeded for valley)
- put this back, plus minor changes from conversations with EFI, Welch, TriLakes
sys/inc/hrms.i
- added reference to NK1 "HRMS" integer value
- must be set to "1" to use reftable; this to prevent unplanned changes to Valley transmissions

jay-advantzware reviewed (commented):
This has hard coded customer id and does not seem to be API driven?

Commit:
62968 EDI - Tri-Lakes (Welch) AND gp export for Shamrock

programs changed:
po/po-gpexp.p - updated export file to support GP requirements for Shamrock
po/po-hrexp.p - fixed issue with scores not resetting by orderline
sys/inc/gp.i - added check of NK1 integer value for GP NK1
sys/ref/sys-ctrl.i - added Shamrock to user list for GP NK1
